packages/loaders/src/styled/StyledSkeleton.ts

```diff
@@ -89,9 +90,13 @@ const retrieveSkeletonGradient = ({
     background-image: linear-gradient(
       ${theme.rtl ? '-45deg' : '45deg'},
       transparent,
-      ${isLight
-        ? getColorV8('chromeHue', 700, theme, 0.4)
-        : rgba(getColorV8('background', 600 /* default shade */, theme)!, 0.6)},
+      ${getColor({
+        theme,
+        hue: 'neutralHue',
+        transparency: theme.opacity[200],
+        light: { shade: isLight ? 500 : 700 },
+        dark: { shade: isLight ? 700 : 500 }
+      })},
```

Comment:
@steue @lucijanblagonic We used to support a [Light Skeleton](https://garden.zendesk.com/components/skeleton#light) for dark surfaces. Now with Dark mode, it will be the inverse shade. 

See in action:
https://6657851b9c7aef2ddba8ea98--zendeskgarden-react-components.netlify.app/?path=/story/packages-loaders-skeleton--skeleton&args=isLight:!true

Hope that works with you.